pkg/web/mobilebuildHandler.go

```diff
@@ -94,3 +94,45 @@ func (bh BuildHandler) GenerateKeys(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 }
+
+func (bh *BuildHandler) AddAsset(rw http.ResponseWriter, req *http.Request) {
+	token := headers.DefaultTokenRetriever(req.Header)
+	buildAsset := &mobile.BuildAsset{}
+	req.ParseMultipartForm(10 * 1000000) //10MB
```

Comment:
Would it be better to use `10 * bytefmt.MEGABYTE` here?

Reply:
nice didn't know about that